src/NServiceBus.Core.Tests/UnitOfWork/UnitOfWorkBehaviorTests.cs

```diff
@@ -206,6 +209,7 @@ public Task End(Exception ex = null)
             }
         }
 
+        [SuppressMessage("Code", "PCR0019:A task-returning method should have a CancellationToken parameter or a parameter implementing ICancellableContext", Justification = "Obsolete.")]
```

Comment:
Could this have been applied to the whole class instead?

Reply:
If this entire class will be removed when we remove unit of work, then yes.